src/engraving/libmscore/slurtie.cpp

```diff
@@ -501,11 +501,7 @@ bool SlurTie::readProperties(XmlReader& e)
 
 void SlurTie::read(XmlReader& e)
 {
-    while (e.readNextStartElement()) {
-        if (!SlurTie::readProperties(e)) {
-            e.unknown();
-        }
-    }
+    Spanner::read(e);
```

Comment:
Probably should just remove this override entirely, there was no need for it and it meant `Slur::readProperties` was never getting called.